tests/native/test_armv7unicorn.py

```diff
@@ -27,8 +27,141 @@
+        "ldr r1, [sp]": b"00109de5",
+        "ldr r1, [sp, #4]": b"04109de5",
+        "ldr r1, [sp], #-5": b"05101de4",
+        "ldr r1, [sp], #5": b"05109de4",
+        "ldr r1, [sp, #-4]!": b"04103de5",
+        "ldr r1, [sp, #4]!": b"0410bde5",
+        "ldr r1, [sp, r2]": b"02109de7",
+        "ldr r1, [sp, -r2]": b"02101de7",
+        "ldr r1, [sp, -r2, lsl #3]": b"82111de7",
+        "ldr r1, [sp, r2, lsl #3]": b"82119de7",
+        "ldr r1, [sp], r2": b"02109de6",
+        "ldr r1, [sp], -r2, lsl #3": b"82111de6",
+        "ldr r1, [sp, r2]!": b"0210bde7",
+        "ldr r1, [sp, -r2, lsl #3]!": b"82113de7",
+        "ldrb r1, [sp]": b"0010dde5",
+        "ldrb r1, [sp, r2]": b"0210dde7",
+        "ldrh r1, [sp]": b"b010dde1",
+        "ldrh r1, [sp, r2]": b"b2109de1",
+        "ldrsb r1, [sp]": b"d010dde1",
+        "ldrsb r1, [sp, r2]": b"d2109de1",
+        "ldrsh r1, [sp]": b"f010dde1",
+        "ldrsh r1, [sp, r2]": b"f2109de1",
+        "lsls r4, r3, 31": b"834fb0e1",
+        "lsls r4, r3, 1": b"8340b0e1",
+        "lsls r4, r3, r2": b"1342b0e1",
+        "lsr r0, r0, r2": b"3002a0e1",
+        "lsr r0, r0, #3": b"a001a0e1",
+        "MLA R1, R2, R3, R4": b"924321e0",
+        "mov r0, 0x0": b"0000a0e3",
+        "mov r0, 0xff000000": b"ff04a0e3",
+        "mov r0, 0x100": b"010ca0e3",
+        "mov r0, 42": b"2a00a0e3",
+        "mov r0, r1": b"0100a0e1",
+        "movs r0, 0": b"0000b0e3",
+        "movs r0, 0xff000000": b"ff04b0e3",
+        "movs r0, 0x100": b"010cb0e3",
+        "movs r0, 0x0e000000": b"0e04b0e3",
+        "movs r0, 42": b"2a00b0e3",
+        "movs r0, r1": b"0100b0e1",
+        "movw r0, 0xffff": b"ff0f0fe3",
+        "movw r0, 0": b"000000e3",
+        "MUL R1, R2": b"910201e0",
+        "MUL R3, R1, R2": b"910203e0",
+        "orr r2, r3, #5": b"052083e3",
+        "orr r2, r3, r4": b"042083e1",
+        "orr r2, r3, r4, LSL #4": b"042283e1",
+        "orr r2, r3": b"032082e1",
+        "orrs r2, r3": b"032092e1",
+        "pop {r1, r2, r3}": b"0e00bde8",
+        "pop {r1}": b"04109de4",
+        "push {r1, r2, r3}": b"0e002de9",
+        "push {r1}": b"04102de5",
+        "RSB r2, r2, #31": b"1f2062e2",
+        "sbc r3, r1, #5": b"0530c1e2",
+        "stm sp, {r1, r2, r3}": b"0e008de8",
+        "stm sp!, {r1, r2, r3}": b"0e00ade8",
+        "stmib   r3, {r2, r4}": b"140083e9",
+        "str SP, [R1]": b"00d081e5",
+        "str R1, [R2, R3]": b"031082e7",
+        "str R1, [R2, R3, LSL #3]": b"831182e7",
+        "str R1, [R2, #3]!": b"0310a2e5",
+        "str R1, [R2], #3": b"031082e4",
+        "sub r3, r1, r2": b"023041e0",
+        "sub r3, r1, #5": b"053041e2",
+        "svc #0": b"000000ef",
+        "UMULLS R1, R2, R1, R2": b"911292e0",
+        "ldr r0, [pc, #-4]": b"04001fe5",
+        "mov r1, r2": b"0210a0e1",
+        "mov r1, r21, r2": b"0230a1e0",
+    },
+    CS_MODE_THUMB: {"add r0, r1, r2": b"01eb0200"},
+}
+
+
+def _ks_assemble(asm, mode=CS_MODE_ARM):
```

Comment:
```suggestion
def _ks_assemble(asm: str, mode=CS_MODE_ARM) -> bytes:
    """Assemble the given string using Keystone using the specified CPU mode."""
    # Explicitly uses late importing so that Keystone will only be imported if this is called.
    # This lets us avoid requiring installation of Keystone for running tests.
```